### PR TITLE
spark batch predict skill

### PR DESCRIPTION
**Pull request by svangapally-cs**

### CODE REVIEW

File: spark-batch-prediction/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
+FROM c12e/spark-template:spark-container-BiyRw4
+
+USER root
+
+COPY --from=redboxoss/scuttle:latest /scuttle /bin/scuttle
+ENV ENVOY_ADMIN_API=http://localhost:15000
+ENV ISTIO_QUIT_API=http://localhost:15020
```
Comment (pkandarpa-cs) [inline spark-batch-prediction/Dockerfile:7]: why is this needed here?

**Review by yreddy-CS — APPROVED** [review]

File: spark-batch-prediction/pom.xml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        </pluginRepository>
+    </pluginRepositories>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.scala-lang</groupId>
+            <artifactId>scala-library</artifactId>
+            <version>2.11.12</version>
+            <scope>provided</scope>
+        </dependency>
```
Comment (jgielstra-cs) [inline spark-batch-prediction/pom.xml:30]: Use properties for spark

File: spark-batch-prediction/pom.xml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-common</artifactId>
+            <version>2.7.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-client</artifactId>
```
Comment (jgielstra-cs) [inline spark-batch-prediction/pom.xml:50]: User properties for hadoop version

**Review by jgielstra-cs — APPROVED** [review]

File: spark-batch-prediction/src/main/python/main.py [hunk 4]
```diff
@@ -0,0 +1,120 @@
+import sys
```
Comment (lkrishna-cs) [inline spark-batch-prediction/src/main/python/main.py:1]: Need copyrights for all our code examples.
